### PR TITLE
Migrate AWS jobs to bootstrap

### PR DESCRIPTION
**Pull request by fejta**
ref #1023 #1021

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1056)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml [hunk 1]
```diff
@@ -36,6 +36,19 @@
...
     name: bootstrap-ci
     suffix:
+    - kubernetes-e2e-aws:
+        job-name: ci-kubernetes-e2e-aws
+        frequency: 'H/5 * * * *' # At least every 30m
+        #trigger-job: 'kubernetes-build'
+    - kubernetes-e2e-aws-release-1.4:
+        job-name: ci-kubernetes-e2e-aws-release-1.4
+        frequency: '@daily'
+        #trigger-job: 'kubernetes-build-1.4'
```
Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: how does the branch works here?

Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: I'll uncomment these after submitting https://github.com/kubernetes/test-infra/pull/1055

Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: ah ok

**Pull request merged**